pkg/v1/tkg/managementcomponents/helper.go

```diff
@@ -22,6 +22,18 @@ const (
 
 // GetTKGPackageConfigValuesFileFromUserConfig returns values file from user configuration
 func GetTKGPackageConfigValuesFileFromUserConfig(managementPackageVersion string, userProviderConfigValues map[string]string) (string, error) {
+	//Temporary hack for TKR source controller paths
```

Comment:
Please added a TODO: and reference the issue if any used to track the work to populate the values correctly from the bom data.